LrcParser/Parser/Lrc/Lines/LrcLyricParser.cs

```diff
@@ -9,8 +9,9 @@ namespace LrcParser.Parser.Lrc.Lines;
 
 public class LrcLyricParser : SingleLineParser<LrcLyric>
 {
-    public override bool CanDecode(string text)
-        => !string.IsNullOrWhiteSpace(text);
+    public override bool CanDecode(string line) =>
+        !string.IsNullOrWhiteSpace(line) &&
+        LrcStartTimeUtils.StartsWithLineTimeTag(line);

```

Comment:
As I remembered, lyric text with no time-tag info can be treated as a lyric.
Feel free to find the evidence if I'm wrong.

Reply:
I think that in that case, the line doesn't follow LRC format, and shouldn't be parsed by the LRC parser. However, we can also change it so that the line time isn't strictly required.

The current `Decode` implementation already supports that and returns empty line start times and time tags with the raw text in that case. Just tell me what you prefer.

Reply:
I prefer to remain the lyric for now.

Maybe in the next step, can:
- Write the validator to find the invalid part.
- Write the utils to fix or drop the lyric after being parsed to the model object (in LrcParser/Model)

Reply:
Ok, sounds good.

Reply:
Fixed in da4c95b43a22af097c2cee2f61bac89919a2b5d0.